src/envdict.jl

```diff
@@ -1,16 +1,16 @@
-function _generate_env_dict(original_env::AbstractDict;
-                            JULIA_DEPOT_PATH)
+function _generate_env_dict(
+    original_env::AbstractDict;
+    JULIA_DEPOT_PATH
+)
     result = Dict{String, String}()
-    if haskey(original_env, "HTTP_PROXY")
-        result["HTTP_PROXY"] = deepcopy(original_env["HTTP_PROXY"])
+
+    for var in ["HTTP_PROXY", "HTTPS_PROXY", "JULIA_PKG_SERVER", "PATH"]
```

Comment:
you put `PATH` here but left out `JULIA_DEPOT_PATH`, but both seem to deepcopy without the check in the original code.

Reply:
oh yeah I see I didn't notice that. Very good